creator-node/src/routes/tracks.js

```diff
@@ -98,7 +105,12 @@ module.exports = function (app) {
     }
 
     req.logger.info(`Time taken in /track_content for full route: ${Date.now() - routeTimeStart}ms for file ${req.fileName}`)
-    return successResponse({ 'track_segments': trackSegments, 'source_file': req.fileName })
+    return successResponse({
+      'transcodedTrackCID': transcodedFilePromResp.multihash,
+      'transcodedTrackUUID': transcodedFilePromResp.fileUUID,
+      'track_segments': trackSegments,
```

Comment:
nit but caseing?

Reply:
So we do camelCase everywhere else and we should actually make trackSegments and sourceFile be camel case too, I just didn't want to rock the boat on this PR

Reply:
makes sense